build-tools/tasks/package-json.js

```diff
@@ -21,6 +21,10 @@ function getComponentsExports() {
     // TypeScript interfaces, can be imported separately from components
     './interfaces': './interfaces.js',
     './contexts/form-field': './contexts/form-field.js',
+    // Public internal components
+    './internal/tooltip-do-not-use': './internal/tooltip-do-not-use/index.js',
+    './test-utils/dom/internal/tooltip': './test-utils/dom/internal/tooltip.js',
+    './test-utils/selectors/internal/tooltip': './test-utils/selectors/internal/tooltip.js',
```

Comment:
Should we add the '_DO_NOT_USE' suffix to the test utils export to avoid accidental imports of the internal tooltip in IDE autocompletes?

Reply:
But would the customer have a case to use it unless they are already using the tooltip?

Reply:
I agree, there shouldn't be a use case. However, there are still some results when searching for imports from internal.
I think the addition of the suffix to the test utils emphasizes the need to avoid it and aligns with the new internal component name.

Reply:
As we talked on slack about: not a hard blocker for me. Approving the PR.